tests/docker/start.irods.ubuntu22.sh

```diff
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 # If testing against locally built packages, uncomment the following line and adjust the version number in the deb files as necessary.
-#apt install -y /irods_packages/irods-database-plugin-postgres_4.3.3-0~jammy_amd64.deb /irods_packages/irods-icommands_4.3.3-0~jammy_amd64.deb /irods_packages/irods-runtime_4.3.3-0~jammy_amd64.deb /irods_packages/irods-server_4.3.3-0~jammy_amd64.deb
+#apt-get install -y /irods_packages/irods-database-plugin-postgres_4.3.3-0~jammy_amd64.deb /irods_packages/irods-icommands_4.3.3-0~jammy_amd64.deb /irods_packages/irods-runtime_4.3.3-0~jammy_amd64.deb /irods_packages/irods-server_4.3.3-0~jammy_amd64.deb
```

Comment:
You typically want to run `apt-get update` before `apt-get install`.

Reply:
done